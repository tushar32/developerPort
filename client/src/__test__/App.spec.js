import React from 'react';
import renderer from 'react-test-renderer';

describe('Counter', () => {
  it('knows that 2 and 2 make 4', () => {
    expect(2 + 2).toBe(5);
  });
});