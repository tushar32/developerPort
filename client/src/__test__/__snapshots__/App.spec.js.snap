// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
<div>
  <p>
    1
  </p>
</div>
`;
